core/src/main/java/bio/terra/pearl/core/model/kit/KitRequestStatus.java

```diff
@@ -5,7 +5,7 @@
 /**
  * High-level, Juniper-centric kit status. Only needs to be specific enough for Juniper to decide how to interact with
  * Pepper. This is not intended to mirror all possible states in Pepper. For that, see
- * {@link bio.terra.pearl.core.service.kit.PepperDSMKitStatus#currentStatus}.
+ * {@link bio.terra.pearl.core.service.kit.PepperKitStatus#currentStatus}.
```

Comment:
This is just a rename refactor. In retrospect, I wish that I had left this change out of this PR. But now that you know about it, it should be pretty easy to filter out.